src/patterns/organisms/navigation/navigation.hbs

```diff
@@ -18,11 +18,13 @@ links:
   <div class="mzp-c-navigation-l-content">
     <div class="mzp-c-navigation-container">
       <button class="mzp-c-navigation-menu-button" type="button" aria-controls="map-c-navigation-items">Menu</button>
-      <div class="mzp-c-navigation-logo"><a href="https://www.mozilla.org/">Mozilla</a></div>
+      {{#if logo}}<div class="mzp-c-navigation-logo"><a href="https://www.mozilla.org/">Mozilla</a></div>{{/if}}
       <div class="mzp-c-navigation-items" id="mzp-c-navigation-items">
```

Comment:
We should probably remove ids like `id="mzp-c-navigation-items"` from the component if we're now using more than one in a single page?

Reply:
✅ ID is now configurable (we need to keep an ID for the ARIA to work properly)